vent/io/devices/base.py

```diff
@@ -55,8 +42,6 @@ def __init__(self, pig: PigpioConnection = None):
         """
         self._pig = pig if pig is not None else PigpioConnection(show_errors=False)
         self._handle = -1
-        if not self.pigpiod_ok:
```

Comment:
why take this out?

Reply:
That was from before - I'm now using a wrapped version of pigpio.pi that throws its own exception (see vent.io.devices.PigpioConnection)